classes/01class/exercises/c01-git02/frdvo/ANSWER.md

```diff
@@ -0,0 +1,22 @@
+# c01-git02
```

Comment:
This file should be named README.md instead of ANSWER.md.. :)

Reply:
sorry mate, In my next PR I will do right!